src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +49,12 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = sessionFactory.openSession()) {
+            Query<Movie> getAllMoviesQuery =
```

Comment:
Minor: As it is dao for movie we can skip word Movies.
```suggestion
            Query<Movie> getAllQuery =
```